pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -33,6 +33,6 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
     license = with licenses; [ lgpl21 gpl2 ];
     homepage = "https://www.displaylink.com/";
-    broken = with kernel; kernelOlder "5" || stdenv.isAarch64;
+    broken = versionOlder kernel.version "4.19" || stdenv.isAarch64;
```

Comment:
This will break for older kernels. Would it be worth it to provide the old version for the old kernels?

Reply:
The number of people that I have found that run DisplayLink on NixOS can be counted on one hand :) I think that would be too much effort. :)